sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -165,7 +162,7 @@ export interface EventHubClientOptions {
  * @class
  * The client is the main point of interaction with Azure Event Hubs service.
  * It offers connection to a specific Event Hub within the Event Hubs namespace and
- * offers operations for sending event data, receiving events, and inspecting the connected Event Hub.
+ * operations for sending event data, receiving events, and inspecting the connected Event Hub.
```

Comment:
There are 2 "and" in this sentence. How about 

> It offers connection to a specific Event Hub within the Event Hubs namespace **along with** operations for sending event data, receiving events, and inspecting the connected Event Hub

?

Reply:
Sure, I don't have a strong opinion on that :)